### PR TITLE
Add actionable stats.

### PR DESCRIPTION
**Pull request by zimathias**

### CODE REVIEW

**Comment by zimathias** [comment]
@babldev

File: pull_alerts.py [hunk 1]
```diff
@@ -65,12 +65,35 @@ def print_all_incidents(group_by_description=False):
...
+        return
+    print("""# Statistics
+| Incidents            | Number |
+| -------------------- | ------ |
+| Total                | {} |
+| Actionable (#a)      | {} |
+| Non Actionable (#na) | {} |
+| Not Tagged           | {} |
+""".format(len(all_incidents),
+           len([i for i in all_incidents if actionable(i)]),
```
Comment (mxr) [inline pull_alerts.py]: Have you considered doing this in one loop through the incidents instead of a loop for each category?

**Review by mxr — APPROVED** [review]

File: pull_alerts.py [hunk 2]
```diff
@@ -109,12 +132,36 @@ def _get_time_window():
     return time_window
 
 
+def actionable(incident):
+    return '#a' in incident.notes
```
Comment (babldev) [inline pull_alerts.py]: I would recommend checking to see if `#a` or `#na` is anywhere in each note, instead of an exact match.

**Comment by babldev** [comment]
+1

File: pull_alerts.py [hunk 3]
```diff
@@ -65,12 +65,42 @@ def print_all_incidents(group_by_description=False):
...
+
+
+def print_stats(all_incidents, include_stats):
+    if not include_stats:
+        return
+    actionable = 0
+    non_actionable = 0
+    not_tagged = 0
+    for i in all_incidents:
+        if actionable(i):
```
Comment (babldev) [inline pull_alerts.py]: Getting `TypeError: 'int' object is not callable`

**Comment by babldev** [comment]
Nice!

```
# Statistics
| Incidents            | Number |
| -------------------- | ------ |
| Total                | 13 |
| Actionable (#a)      | 4 |
| Non Actionable (#na) | 7 |
| Not Tagged           | 2 |
```

**Pull request merged**